src/qibolab/instruments/dummy.py

```diff
@@ -43,19 +43,19 @@ class DummyInstrument(Controller):
     sampling_rate = 1
 
     def connect(self):
-        log.info("Connecting to dummy instrument.")
+        log.info(f"Connecting to {self.name} instrument.")
 
     def setup(self, *args, **kwargs):
-        log.info("Setting up dummy instrument.")
+        log.info(f"Setting up {self.name} instrument.")
 
     def start(self):
-        log.info("Starting dummy instrument.")
+        log.info(f"Starting {self.name} instrument.")
 
     def stop(self):
-        log.info("Stopping dummy instrument.")
+        log.info(f"Stopping {self.name} instrument.")
 
     def disconnect(self):
-        log.info("Disconnecting dummy instrument.")
+        log.info(f"Disconnecting {self.name} instrument.")
```

Comment:
Why do we need again these `{self.name}`? Was it because there were two dummies?

At the moment, `self.name` is always:
https://github.com/qiboteam/qibolab/blob/b7a7d7a0dc1490d14f34320abf755e247b7e1189/src/qibolab/dummy.py#L9

Reply:
However, it doesn't matter, it is fine even like this.

Reply:
Yup, I left it just in case we change the names or came up with dummy2 again for other reasons. I can remove it, no problem.